# Alignments
Nucleotide alignments of fungal nucleotide sequences created as parts of new species descriptions. Sequences were aligned using MAFFT (Katoh & Standley, 2013) and trimmed with the use of trimAl with -automated1 algorithm (Capella-Gutierrez et al., 2009).

# Files
E.ferrotolerans_al.fasta - file containing trimmed alignment of ITS sequences of _Entomortierella parvispora_ complex as understood by Telagathoti et al., 2022.
E.ferrotolerans_data_description.csv - file containing GenBank accession IDs of data used, strain IDs and typification corresponding to the sequences names in the alignment.

# Literature
Capella-Gutierrez S, Silla-Martinez JM, Gabaldon T. trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses. Bioinformatics. 2009 Aug 1;25(15):1972–3. 
Katoh K, Standley DM. MAFFT Multiple Sequence Alignment Software Version 7: Improvements in Performance and Usability. Molecular Biology and Evolution. 2013 Apr 1;30(4):772–80. 
Telagathoti A, Probst M, Mandolini E, Peintner U. Mortierellaceae from subalpine and alpine habitats: new species of Entomortierella, Linnemannia, Mortierella, Podila and Tyroliella gen. nov. Stud Mycol. 2022 Sep;103:25-58. doi: 10.3114/sim.2022.103.02. Epub 2022 Oct 18. PMID: 37342154; PMCID: PMC10277274.
